Libraries/Text/Text.js

```diff
@@ -177,6 +177,10 @@ const Text = React.createClass({
      */
     testID: PropTypes.string,
     /**
+     * If true, plays system sound on touch
```

Comment:
Wait, why `<Text>`? Text has nothing to do with sound. A property of `<Touchable*>` would be enough.